src/org/parosproxy/paros/extension/encoder/Encoder.java

```diff
@@ -154,6 +155,82 @@ public String getIllegalUTF8Encode(String msg, int bytes) {
 		
 		return sbResult.toString();
 	}
+
+
+	public String getUnicodeEncode(String str) {
+        str = (str == null ? "" : str);
+        String tmp;
+        StringBuffer sb = new StringBuffer(1000);
+        char c;
+        int i, j;
+        sb.setLength(0);
+        for (i = 0; i < str.length(); i++)
+        {

```

Comment:
This code does not seem to be formatted similarly to the original code in the file. (Curly braces on starting lines of control blocks, indent, etc.)